### PR TITLE
feature-check-mark

### PR DESCRIPTION
**Pull request by Aion152**
added checks to see who has won a board
added ability to mark winning boards

### CODE REVIEW

**Comment by MentatBashar** [comment]
Commmit messages are bad. Closing pull request.

**Comment by Aion152** [comment]
> 

You are bad. I am going to close you.

File: caw.c [hunk 1]
```diff
@@ -430,6 +431,57 @@ bool within_cell(double x, double y, int i, int j, int t)
     return false;
 }
 
+void check_nac_board(int i, int j, int mark)
+{
+  printf("%d", mark);
```
Comment (MentatBashar) [inline caw.c:436]: You have left in your debug statements ._.

**Review by MentatBashar — APPROVED** [review]
Looks good. Will be refactored at a later date (i.e. never). Danke.

**Pull request merged**